Dockerfile

```diff
@@ -13,4 +13,6 @@ RUN [ "npm", "run", "postinstall" ]
 
 WORKDIR /usr/src/app/packages/vc-http-api
 
-CMD [ "npm", "run", "start" ]
+RUN [ "npm", "run", "build" ]
```

Comment:
@OR13 this change significantly reduces the cold boot time from 20s down to 3s. 

The build command was running every-time the container was woken up